ush/fire_emiss_tools.py

```diff
@@ -103,10 +135,34 @@ def estimate_fire_duration(intp_avail_hours, intp_dir, fcst_dates, current_day,
     return(te)
 
 def save_fire_dur(cols, rows, te):
+    """
+    ???
+
+    cols: ???
+    rows: ???
+    te: ???

```

Comment:
@haiqinli and @MatthewPyle-NOAA what do each of these parameters mean and what does this function do?